src/ceph-detect-init/CMakeLists.txt

```diff
@@ -1,12 +1,9 @@
-set(CEPH_DETECT_INIT_VIRTUALENV ${CEPH_BUILD_VIRTUALENV}/ceph-detect-init-virtualenv)
+include(Distutils)
+distutils_install_module(ceph_detect_init)
 
-add_custom_target(ceph-detect-init
+add_custom_target(ceph-detect-init-flake8
   COMMAND
-  ${CMAKE_SOURCE_DIR}/src/tools/setup-virtualenv.sh ${CEPH_DETECT_INIT_VIRTUALENV} &&
-  ${CEPH_DETECT_INIT_VIRTUALENV}/bin/pip install --no-index --use-wheel --find-links=file:${CMAKE_SOURCE_DIR}/src/ceph-detect-init/wheelhouse -e .
+  flake8 ceph_detect_init tests
   WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/src/ceph-detect-init
-  COMMENT "ceph-detect-init is being created")
-add_dependencies(tests ceph-detect-init)
-
-include(Distutils)
-distutils_install_module(ceph_detect_init)
+  COMMENT "syntax check")
+add_dependencies(ceph_detect_init ceph-detect-init-flake8)
```

Comment:
we don't have a target named `ceph_detect_init` yet. `ceph_detect_init` is used to create another target named `ceph_detect_init-clone`  for copying the source files into the build directory.